Stratis.Bitcoin.Features.Wallet/WalletExtensions.cs

```diff
@@ -11,7 +11,7 @@ public static class WalletExtensions
         /// <param name="chain">The chain.</param>        
         public static bool IsDownloaded(this ConcurrentChain chain)
         {
-            return chain.Tip.Header.BlockTime.ToUnixTimeSeconds() > (DateTimeOffset.Now.ToUnixTimeSeconds() - TimeSpan.FromHours(1).TotalSeconds);
+            return chain.Tip.Header.BlockTime.ToUnixTimeSeconds() > (DateTimeOffset.UtcNow.ToUnixTimeSeconds() - TimeSpan.FromHours(1).TotalSeconds);
```

Comment:
Hmm not the best but I guess its an exception

Reply:
one of 2 or 3 exceptions only, no problem to have it though